cmd/evm-middleware/main.go

```
@@ -0,0 +1,72 @@
+func main() {
+	jwtSecret := ""
+	if len(os.Args) == 2 {
+		jwtSecret = os.Args[1]
+	}
+
+	config := &grpcproxy.Config{
+		DefaultTimeout: 5 * time.Second,
+		MaxRequestSize: bufSize,
+	}
+
+	listener, err := net.Listen("tcp4", "0.0.0.0:40041")
```

golangci-lint (1.62.2)

[medium] G102: Binds to all network interfaces

(gosec)